src/Entrust/EntrustServiceProvider.php

```diff
@@ -50,6 +53,41 @@ public function register()
     }
 
     /**
+     * Register the blade directives
+     *
+     * @return void
+     */
+    private function bladeDirectives()
+    {
+        // Call to Entrust::hasRole
+        Blade::directive('role', function($expression) {
+            return "<?php if (\\Entrust::hasRole{$expression}) : ?>";
+        });
+
+        Blade::directive('endrole', function($expression) {
+            return "<?php endif; // Entrust::hasRole ?>";
+        });
+
+        // Call to Entrust::can
```

Comment:
Would it be possible to have "may" instead of "can", please? "User 1 can do sth" and "User 1 may do sth" sound similar.